defaults/main.yml

```diff
@@ -59,7 +59,7 @@ logstash_major_ver: '5.x'  # Define major version to install
 # Defines the miniumum amount of memory (in MB) required to effectively run Logstash
 logstash_min_memory_required: 1024
 
-logstash_minor_ver: '1:5.2.0-1'  # Define minor version to install
```

Comment:
I have done the implementation of APT preferences with Pin-priority. 
Also added `force: yes` to apt install due to the need to get it working properly always (upgrading/downgrading)

Testing now downgrading also works fine, example: 

output for policy:
```shell
apt-cache policy logstash
logstash:
  Instalados: 1:5.4.1-1
  Candidato:  1:5.3.3-1
  Pin del paquete: 1:5.3.3-1
```

Install downgrades correctly. 
This will also ensure the logstash version will stay during `apt upgrades`, very important as I have noticed when the logstash apt package upgrades without ansible, the plugins installed get missed so logstash fails if you don't use this ansible-logstash role to upgrade.